app/templates/partials/responses/radio.html

```diff
@@ -1,8 +1,9 @@
-<div class="field">
-  {% for option in response.options %}
-    <div class="field__item">
-      <input class="input input--radio" type="radio" name="{{response.name}}" id="{{option.id}}" value="{{option.value}}" title="{{option.label}}">
-      <label class="label label--inline" for="{{option.id}}"><span class="label__inner">{{option.label}}</span></label>
-    </div>
-  {% endfor %}
-</div>
+{{form.label(response)}}
+{% for option in response.options %}
+  <div class="field__item">
+    <input class="input input--radio" type="radio" name="{{response.id}}" id="{{option.id}}" value="{{option.value}}" title="{{option.value}}">&nbsp;&nbsp;{{option.value}}
+    <label class="label label--inline" for="{{option.id}}"><span class="label__inner">{{option.label}}</span></label>
+  </div>
+{% endfor %}
+<input type="hidden" name="{{response.id}}" value="">
```

Comment:
Is this needed? You're already sending this as the `name` attribute of the `input` itself.